UnitTests/GitCommands.Tests/Submodules/SubmoduleStatusProviderTests.cs

```diff
@@ -64,7 +64,8 @@ public void TearDown()
             [Test]
             public async Task UpdateSubmoduleStructure_valid_result_for_top_module()
             {
-                var result = await SubmoduleTestHelpers.UpdateSubmoduleStructureAndWaitForResultAsync(_provider, _repo1Module);
+                var result = await SubmoduleTestHelpers.UpdateSubmoduleStructureAndWaitForResultAsync(_provider, _repo1Module).ConfigureAwait(false);
+                result.Should().NotBeNull();
 
                 result.TopProject.Path.Should().Be(_repo1Module.WorkingDir);
                 result.SuperProject.Should().Be(null);
```

Comment:
nit: The pattern here was: Empty line before the assertions.
We can make the not-null assertion explicit but then for all call sites.

Reply:
> ... but then for all call sites.

Already there for all call sites. The diff display in GH made me think some were missing.